Analysis/Tasks/PWGDQ/tableReader.cxx

```diff
@@ -531,6 +557,31 @@ struct DQTableReader {
         }
       }
     } // end loop over muon track pairs
+
+    for (auto& [trackBarrel, trackMuon] : combinations(tracks, muons)) {
+      twoTrackFilter = trackBarrel.isBarrelSelected() & trackMuon.isMuonSelected() & fTwoTrackFilterMask;
```

Comment:
The selection filter bits for electrons are obviously different from those on muons. In particular, at the moment the muons selection task is applying just a single cut, while for the electrons multiple ones are possible. So when you do a bitwise AND you only request that the muon cut is fulfilled together with the first electron cut. All the rest of electron cuts are rejected.
There are many ways to combine electron and muon cuts, so one has to think of a strategy if you will want at some point to combine different electron and muon selections. Within the current model of this "example workflow", as i suggested in the previous comment you should combine the muon single cut with all the requested electron cuts. For this, the twoTrackFilter should be computed like this:
twoTrackFilter = (trackBarrel.isBarrelSelected() & fTwoTrackFilterMask) && trackMuon.isMuonSelected();

Reply:
Oh, I get it! Sorry, I misunderstood what you were saying before.